services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(tickets: list[dict], username: str, date: str = None) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(
+            user=user
+        )
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket in tickets:
+            movie_session = MovieSession.objects.get(

```

Comment:
Getting objects here causes unnecessary database load. Use `movie_session_id` argument to pass id directly.